packages/dds/sequence/src/intervalCollection.ts

```diff
@@ -193,9 +195,32 @@ export interface IIntervalHelpers<TInterval extends ISerializableInterval> {
 		intervalType: IntervalType,
 		op?: ISequencedDocumentMessage,
 		fromSnapshot?: boolean,
+		stickiness?: IntervalStickiness,
 	): TInterval;
 }
 
+export enum IntervalStickiness {
```

Comment:
i'd prefer if we didn't use and enum (see how we do ops) and didn't use flags.

Reply:
neither flags or enums are well supported in typescript/javascript, and don't really add value here, its just added complexity

Reply:
i would apply this feedback to all enums added in this pr

Reply:
I wasn't sure if you meant to use a type like `"forward" | "backward"` or something similar to what is done for `MergeTreeDeltaType`. I've implemented something similar to the latter in https://github.com/microsoft/FluidFramework/pull/15247/commits/a3b0332b6dfb262e23ed30e18aafa22d4df96ddf as that's what seems to be done in the ops file. It's effectively a 15 second change to switch between these, so if I've misinterpreted this it should be easy to fix.

For the former case, I can see not having to import the enum as a bonus, but for the latter case, I'm under the impression that the enum option will compile down to effectively the same thing, so I will probably ask about the difference in parking lot